component/common/net/server.go

```diff
@@ -58,6 +63,28 @@ func NewTargetServer(logger log.Logger, metricsNamespace string, reg prometheus.
 	return ts, nil
 }
 
+func (ts *TargetServer) registerManagementAPI(router *mux.Router) {
```

Comment:
Since we have some form of config management this may need a different name to be easily differentiated.